cloud-consumption/src/main/java/com/sequenceiq/consumption/endpoint/ConsumptionController.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.consumption.endpoint;
+
+import javax.transaction.Transactional;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+
+import com.sequenceiq.cloudbreak.structuredevent.rest.annotation.AccountEntityType;
+import com.sequenceiq.consumption.api.v1.consumption.endpoint.ConsumptionEndpoint;
+import com.sequenceiq.consumption.api.v1.consumption.model.request.StorageConsumptionScheduleRequest;
+import com.sequenceiq.consumption.api.v1.consumption.model.request.StorageConsumptionUnscheduleRequest;
+import com.sequenceiq.consumption.domain.Consumption;
+import com.sequenceiq.consumption.service.ConsumptionService;
+
+@Controller
+@Transactional(Transactional.TxType.NEVER)
+@AccountEntityType(Consumption.class)
+public class ConsumptionController implements ConsumptionEndpoint {
```

Comment:
missing authorization.
also please add `EnforceAuthorizationAnnotationsTest` to the consumption service, other modules has it